dz_s5/zadacha_2.py

```diff
@@ -0,0 +1,14 @@
+# Напишите однострочный генератор словаря, который принимает на вход
+# три списка одинаковой длины: имена str, ставка int, премия str
+# с указанием процентов вида “10.25%”. В результате получаем словарь
+# с именем в качестве ключа и суммой премии в качестве значения.
+# Сумма рассчитывается как ставка умноженная на процент премии
+
+
+list_names = ["ivan", "petr", "yra"]
+list_salary = [30_000, 25_000, 28_000]
+list_award = ["10.25%", "11.25%", "10.05%"]
+
+dict_award = {name: salary * (float(award.rstrip('%')) / 100)
```

Comment:
Абсолютно верное решение.
```suggestion
dict_award = {name: salary * float(award.rstrip('%')) / 100
```